x-pack/test/security_solution_endpoint/page_objects/hosts_page.ts

```diff
@@ -105,5 +107,46 @@ export function SecurityHostsPageProvider({ getService, getPageObjects }: FtrPro
       await deleteMetadataStream(getService);
       await deleteTelemetryStream(getService);
     },
+    /**
+     * Runs Nodes Events
+     */
+    async runNodeEvents(expectedData: string[]) {
+      await testSubjects.exists('resolver:submenu:button', { timeout: 400 });
```

Comment:
For the current tests where only one node will have events this works.  Is there a way to make this more selective per node?  If more than one node has events this could select the wrong submenu.  I think I can thumb the pr either way but this might be an issue in future tests.

Reply:
It will not select wrong sub-menu. Script selects all sub-menus and opens them at same time, and they going in the order